### DISCUSSION TITLE
Would like to see syphon on the list of experimental apps

### DISCUSSION THREAD

**Discussion by FillingTheVoidOnYT** [syphon-org/syphon#725] [Ideas]
Would like to see syphon on the list of experimental apps when opening matrix.to links to rooms. This would be convenient when joining rooms, eventually it can transition into regular apps like FluffyChat is listed.

**Comment by ereio** [comment]
Good idea, I forgot they had an experimental section now @0x1a8510f2

**Reply by 0x1a8510f2** [reply]
Cool! I'll get to it soon :D

**Reply by 0x1a8510f2** [reply]
I've opened a PR :blush: https://github.com/matrix-org/matrix.to/pull/280

Let me know if I should tweak any of the metadata.

As a side note, do we support deep-linking? I omitted that bit for now but I'll need to update it if we do.

**Reply by 0x1a8510f2** [reply]
Wow, that was quick. Looks like we made it!
(Note that this is only the [CI preview](https://017b0ce2.matrix-to-asy.pages.dev/#/#matrix:matrix.org) and the site still needs to be updated, but the PR has been merged!)

![image](https://user-images.githubusercontent.com/83190128/190436840-036ff7cb-084e-419d-ab61-a628db9866ce.png)
